ansible/roles/kstest-master/templates/run_tests.sh.j2

```diff
@@ -183,7 +184,7 @@ fi
 ### Run the test
 
 RESULT_LOG="${RESULT_PATH}/${LOG_FILENAME}"
-scripts/run_kickstart_tests.sh -i ${BOOT_ISO} -k 1 ${UPDATES_IMAGE_ARG} ${ADDITIONAL_BOOT_OPTIONS_ARG} ${TEST_TYPE_ARG} ${SKIP_TEST_TYPES_ARG} ${PLATFORM_ARG} ${OVERRIDE_ARG} ${KSTESTS} 2>&1 | tee ${RESULT_LOG}
+eval scripts/run_kickstart_tests.sh -i ${BOOT_ISO} -k 1 ${UPDATES_IMAGE_ARG} ${ADDITIONAL_BOOT_OPTIONS_ARG} ${TEST_TYPE_ARG} ${SKIP_TEST_TYPES_ARG} ${PLATFORM_ARG} ${OVERRIDE_ARG} ${KSTESTS} 2>&1 | tee ${RESULT_LOG}
```

Comment:
Is this really required? If the spaces are the problem, what about just add "" around the variables?

Reply:
Yeah, I tried that, but it is not not working because empty value (eg ADDITIONAL_BOOT_OPTIONS_ARG="") would produce an empty argument ('') in this case.

Reply:
Please add a comment with explanation why eval and ideally with the link on detailed explanation.